packages/core/lib/commands/config.js

```diff
@@ -21,6 +21,10 @@ const command = {
       {
         option: "set",
         description: "Set a Truffle config option value."
+      },
+      {
+        option: "list",
+        description: "List a Truffle config option values.",
```

Comment:
So I think this should be "List all Truffle config values." That's what this feature is meant to do, right? List all the stored Truffle config values?

Reply:
You're right. Done.